src/Domain/Entities/Booking.cs

```diff
@@ -14,11 +15,9 @@ public class Booking : BaseEntity<Guid>
         /* EF Relation */
```

Comment:
You should change sequence of properties like this

public class Booking : BaseEntity<Guid>
    {
        public DateTime BookingStart { get; set; }
        public DateTime BookingEnd { get; set; }
        public bool IsRecurring { get; set; }
        public string Frequancy { get; set; }
        public bool IsActive { get; set; }
        
        /* EF Relation */
        public Guid UserId { get; set; }
        public AppUser User { get; set; }

        public Guid? ParkingPlaceId { get; set; }
        public ParkingPlace ParkingPlace { get; set; }

        public Guid WorkPlaceId { get; set; }
        public WorkPlace WorkPlace { get; set; }
    }

Reply:
this comment applies for all entities